client/signup/jetpack-connect/index.jsx

```diff
@@ -197,10 +201,79 @@ const JetpackConnectMain = React.createClass( {
 					<p>{ this.translate( 'Your jetpack seems to be inactive! We need to send you to your WordPress install so you can activate it before completing the connection.' ) }</p>
```

Comment:
Not related to this particular PR, either. Maybe Jetpack should be spelled with a capital J